src/main/kotlin/io/polygon/kotlin/sdk/websocket/PolygonWebSocketClient.kt

```diff
@@ -47,11 +65,11 @@ class PolygonWebSocketClient
 @JvmOverloads
 constructor(
     private val apiKey: String,
-    val cluster: PolygonWebSocketCluster,
+    private val feed: Feed = Feed.RealTime, // defaults to RealTime feed if not provided
+    private val market: Market = Market.Stocks, // defaults to Stocks market if not provided
```

Comment:
nit: these comments aren't necessary, the default is already explicitly there. 

```suggestion
    private val feed: Feed = Feed.RealTime,
    private val market: Market = Market.Stocks,
```